src/components/LoginForm/FormAdmin.tsx

```diff
@@ -0,0 +1,83 @@
+import { useForm } from 'react-hook-form';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useMutation } from 'react-query';
+import { AxiosResponse } from 'axios';
+import { useNavigate } from 'react-router-dom';
+import { toast } from 'react-toastify';
+import { Button, ForgetPassword, Form, InputForm, TitleLogin } from './Login.styled';
+import loginSchema from '../../utils';
+
+import { authRoutes } from '../../services';
+
+interface FormData {
+  email: string;
+  password: string;
+}
+
+interface ReqError extends AxiosResponse {
+  response: {
+    data: {
+      error: boolean;
+      data: {
+        message: string;
+      };
+    };
+  };
+}
+
+const LoginForm = () => {
+  const navigate = useNavigate();
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+  } = useForm<FormData>({ resolver: zodResolver(loginSchema), mode: 'onBlur' });
+
+  const { isLoading, mutate, error } = useMutation((data: FormData) => authRoutes.login(data), {
+    onSuccess: (data) => {
+      localStorage.setItem('user', JSON.stringify(data.data.data.user));
+      if (data.data.data.user.isAdmin) {
+        navigate('/admin');
+      } else {
+        toast.error('This page just for Admin');
+        navigate('/admin');
```

Comment:
I think here you shouldn't navigate to `/admin` just show the alert

Reply:
coz iam handleing delete token in the provider checkAuthAdmin

Reply:
I mean in line 43.
You should show to users the `toast alert`